api/src/main/java/org/pmiops/workbench/cdr/dao/DSDataDictionaryDao.java

```diff
@@ -0,0 +1,8 @@
+package org.pmiops.workbench.cdr.dao;
+
+import org.pmiops.workbench.cdr.model.DbDSDataDictionary;
+import org.springframework.data.repository.CrudRepository;
+
+public interface DSDataDictionaryDao extends CrudRepository<DbDSDataDictionary, Long> {
+  DbDSDataDictionary findDbDSDataDictionaryByFieldNameAndDomain(String field_name, String domain);
```

Comment:
You can shorten this method name.
```suggestion
  DbDSDataDictionary findByFieldNameAndDomain(String field_name, String domain);
```

Reply:
Done